sequencer/src/persistence/sql.rs

```diff
@@ -1805,6 +1806,44 @@ impl SequencerPersistence for Persistence {
         tx.commit().await
     }
 
+    async fn add_state_cert(
+        &self,
+        state_cert: LightClientStateUpdateCertificate<SeqTypes>,
+    ) -> anyhow::Result<()> {
+        let state_cert_bytes = bincode::serialize(&state_cert)
+            .context("serializing light client state update certificate")?;
+
+        let mut tx = self.db.write().await?;
+        tx.upsert(
+            "state_cert",
+            ["epoch", "state_cert"],
+            ["epoch"],
+            [(state_cert.epoch.u64() as i64, state_cert_bytes)],
+        )
+        .await?;
+        tx.commit().await
+    }
+
+    async fn load_state_cert(&self) -> anyhow::Result<LightClientStateUpdateCertificate<SeqTypes>> {
+        match self
+            .db
+            .read()
+            .await?
+            .fetch_one("SELECT state_cert, MAX(epoch) from state_cert GROUP BY epoch")
```

Comment:
think you want to use fetch_optional here

Reply:
The fetch_one will return an error if there is no row

Reply:
Use `fetch_optional` to return an Option: [60ddcad](https://github.com/EspressoSystems/espresso-network/pull/2762/commits/60ddcada8e0f37d832f2a9322726839109c12964)